packages/babel-plugin-transform-block-scoping/src/index.js

```diff
@@ -545,7 +543,7 @@ class BlockScoping {
     this.hoistVarDeclarations();
 
     // turn outsideLetReferences into an array
-    const args = values(outsideRefs).map(id => t.cloneNode(id));
+    const args = [...outsideRefs.values()].map(node => t.cloneNode(node));
```

Comment:
Tip: `Array.from` supports a second parameter for this use case:

```js
Array.from(outsideRefs.values(), node => t.cloneNode(node));
```
Bit cleaner (and probably faster? iterates once?)

Reply:
Neat, thanks :)  Change applied.